docs/contributing.rst

```diff
@@ -46,6 +46,176 @@ a human. All code style enforcement must be automated to avoid bikeshedding
+    features:
+    - |
+      graphene: Adds support for ``graphene>=2``. `See the graphql documentation <https://ddtrace.readthedocs.io/en/1.6.0/integrations.html#graphql>`_
+      for more information.
+
+* Upgrade: Enhanced functionality or if dependencies are upgraded. Also used for if components are removed. Usually includes instruction or recommendation to user in regards to how to adjust to the new change. For example::
+
+    upgrade:
+    - |
+      tracing: Use ``Span.set_tag_str()`` instead of ``Span.set_tag()`` when the tag value is a
+      text type as a performance optimization in manual instrumentation.
+
+* Deprecations: Warning of a component being removed from the public API in the future. For example::
+
+    deprecations:
+    - |
+      tracing: ``ddtrace.Span.meta`` has been deprecated. Use ``ddtrace.Span.get_tag`` and ``ddtrace.Span.set_tag`` instead.
+
+* Fixes: Bug fixes. For example::
+
+    fixes:
+    - |
+      django: Fixes an issue where a manually set ``django.request`` span resource would get overwritten by the integration.
+
+* Other: Any change which does not fall into any of the above categories. For example::
+
+    other:
+    - |
+      docs: Adds documentation on how to use Gunicorn with the ``gevent`` worker class.
+
+* Prelude: Not required for every change. Required for major changes such as a new component or new feature which would benefit the user by providing additional context or theme. For example::
+
+    prelude: >
+      dynamic instrumentation: Dynamic instrumentation allows instrumenting a running service dynamically
+      to extract runtime information that could be useful for, e.g., debugging
+      purposes, or to add extra metrics without having to make code changes and
+      re-deploy the service. See https://ddtrace.readthedocs.io/en/1.6.0/configuration.html
+      for more details.
+    features:
+    - |
+      dynamic instrumentation: Introduces the public interface for the dynamic instrumentation service. See
+      https://ddtrace.readthedocs.io/en/1.6.0/configuration.html for more details.
+
+Release Note Formatting
++++++++++++++++++++++++
+
+In general, a release note entry should follow the following format::
+
+  ---
+  <section>:
+    - |
+      scope: note
+
+Scope
+~~~~~
+
+This is a one-word scope, which is ideally the name of the library component, sub-component or integration
+that is impacted by this change. This should not be capitalized unless it is an acronym.
+
+To ensure consistency in component naming, the convention in referring to components is as follows:
+
+* Tracer: ``tracing``
+* Profiler: ``profiling``
+* Application security monitoring: ``ASM``
+* Dynamic instrumentation: ``dynamic instrumentation``
+* CI Visibility: ``CI visibility``
+* Integrations: ``integration_name``
+
+Note
+~~~~
+
+The note is a brief description of the change. It should follow sentence-case capitalization.
+The note should also follow valid restructured text (RST) formatting.
+
```

Comment:
Should we just say to see the template release note since this information is duplicated there?